new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -137,4 +232,162 @@ private static void writeToFile(Object obj, String resultDir, String fileName) t
+    V1Pod pvPod = null;
+    try {
+      // create a temporary pod to get access to the interested persistent volume
+      pvPod = setupPVPod(namespace, hostPath);
+      copyDirectoryFromPod(pvPod, hostPath, destinationPath);
+    } finally {
+      // remove the temporary pod
+      if (pvPod != null) {
+        cleanupPVPod(namespace);
+      }
+    }
+  }
+
+  /**
+   * Creates temporary pod with persistent volume claim and persistent volume using host path.
+   *
+   * @param namespace name of the namespace
+   * @param hostPath host path from ineterested persistent volume
+   * @return V1Pod pod object
+   * @throws ApiException when create pod fails
+   */
+  private static V1Pod setupPVPod(String namespace, String hostPath) throws ApiException {
+
+    ConditionFactory withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(5, SECONDS)
+        .atMost(1, MINUTES).await();
+
+    // create a pvc and pv to get access to the host path of the target pv
+    final String pvcName = "pv-pod-pvc-" + namespace;
+    final String pvName = "pv-pod-pv-" + namespace;
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(namespace + "-weblogic-domain-storage-class")
+            .putCapacityItem("storage", Quantity.fromString("2Gi"))
+            .persistentVolumeReclaimPolicy("Recycle")
+            .hostPath(new V1HostPathVolumeSource().path(hostPath)))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvName)
+            .build());
+    TestActions.createPersistentVolume(v1pv);
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .volumeName(pvName)
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(namespace + "-weblogic-domain-storage-class")
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("2Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(namespace)
+            .build());
+    TestActions.createPersistentVolumeClaim(v1pvc);
+
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pv to be bound, "
+                + "(elapsed time {0} , remaining time {1}",
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(isPersistentVolumeInState(pvName, "Bound"));
+
+    final String podName = "pv-pod-" + namespace;
+    V1Pod podBody = new V1Pod()
+        .spec(new V1PodSpec()
+            .containers(Arrays.asList(
+                new V1Container()
+                    .name("pv-container")
+                    .image("nginx")
```

Comment:
don't use nginx... use oraclelinux:7-slim

Reply:
The copy api needs tar utility in the one-off pod, oraclelinux:7-slim doesn't have it. Instead using a oraclelinux:latest image